src/pyscript/plugins/run.py

```diff
@@ -30,6 +30,11 @@ class FolderBasedHTTPRequestHandler(SimpleHTTPRequestHandler):
         def __init__(self, *args, **kwargs):
             super().__init__(*args, directory=folder, **kwargs)
 
+        def end_headers(self):
+            self.send_header("Cross-Origin-Opener-Policy", "same-origin")
+            self.send_header("Cross-Origin-Embedder-Policy", "require-corp")
```

Comment:
for SharedArrayBuffer to work also on Safari you need these 3 headers:
https://github.com/WebReflection/mini-coi/blob/main/mini-coi.js#L21-L23

```
self.send_header('Cross-Origin-Opener-Policy', 'same-origin');
self.send_header('Cross-Origin-Embedder-Policy', 'require-corp');
self.send_header('Cross-Origin-Resource-Policy', 'cross-origin');
```

if you set COEP but no CORP Safari complains and so might others.

Reply:
Awesome. TY!

Reply:
you need to push the change though :D